### PR TITLE
Python

### PR DESCRIPTION
**Pull request by snehanshup**
name changed pull request

### CODE REVIEW

**Comment by snehanshup** [comment]
looks good to me

File: intro.py [hunk 1]
```diff
@@ -1 +1 @@
-print "(My name is NK)"
+print "(My name is snehanshu)"
```
Comment (nkdash001) [inline intro.py]: you need to change this to my name

**Review by nkdash001 — CHANGES_REQUESTED** [review]
change name to my name

**Review by snehanshup — COMMENTED** [review]
nice

**Review by nkdash001 — APPROVED** [review]

**Review by nkdash001 — APPROVED** [review]

**Comment by snehanshup** [comment]
ok done

File: intro.py (hunk 1 above)
Comment (snehanshup) [reply intro.py]: done

**Review by snehanshup — COMMENTED** [review]
alright fine  LGTM

**Comment by snehanshup** [comment]
closed

**Pull request closed without merging**